### PR TITLE
VisualC-GDK: fix tests on Xbox

### PR DESCRIPTION
**Pull request by code-disaster**
Tests included in VisualC-GDK now default to the GPU renderer, but crash on Xbox because the swap chain size isn't set, causing the backbuffer texture creation to fail.

The second change just enables the GameInput joystick driver in `testcontroller`, which is the only system driver initialized on Xbox right now.

### CODE REVIEW

File: test/testcontroller.c [hunk 1]
```diff
@@ -2005,6 +2005,10 @@ int main(int argc, char *argv[])
     SDL_SetHint(SDL_HINT_JOYSTICK_ALLOW_BACKGROUND_EVENTS, "1");
     SDL_SetHint(SDL_HINT_JOYSTICK_LINUX_DEADZONES, "1");
 
+#if SDL_PLATFORM_GDK
+    SDL_SetHint(SDL_HINT_JOYSTICK_GAMEINPUT, "1");
```
Comment (slouken) [inline test/testcontroller.c:2009]: We should probably enable this by default in SDL for XBox, instead of doing it here.

**Review by flibitijibibo — APPROVED** [review]
GPU changes lgtm

File: test/testcontroller.c (hunk 1 above)
Comment (code-disaster) [reply test/testcontroller.c]: Pushed a change which defaults the hint to "1" for all GDK platforms, including Windows.

**Comment by slouken** [comment]
Looks good, thanks!

**Pull request merged**